### PR TITLE
Run django migrations before running server

### PR DESCRIPTION
**Pull request by jjnesbitt**
My only remaining concern is if we should have some check in place to make sure we're not running an old image version? I was thinking of using the migration system to check for that, although I don't know how exactly.

### CODE REVIEW

File: .github/workflows/custom_docker_builds.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
           - docker-image: ./images/build-timing-processor
             image-tags: ghcr.io/spack/build-timing-processor:0.0.4
           - docker-image: ./analytics
-            image-tags: ghcr.io/spack/django:0.0.1
+            image-tags: ghcr.io/spack/django:0.0.2
```
Comment (danlamanna) [inline .github/workflows/custom_docker_builds.yml:51]: Don't the references need to be updated for this as well?

Comment (danlamanna) [reply .github/workflows/custom_docker_builds.yml:51]: The celery runners also override this command and probably need to run migrate as well. What do you think?

Comment (jjnesbitt) [reply .github/workflows/custom_docker_builds.yml:51]: Good catch, I totally forgot about the other references. Regarding the command overwriting in celery as well, would it make sense to move the call to `migrate` into the entrypoint? Essentially using the entrypoint script pattern shown [here](https://stackoverflow.com/a/41518225).

Comment (jjnesbitt) [reply .github/workflows/custom_docker_builds.yml:51]: (testing that pattern locally seems to work as intended)

Comment (danlamanna) [reply .github/workflows/custom_docker_builds.yml:51]: Sounds good to me.

File: analytics/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Always run migrate when invoking this image
+python manage.py migrate
+
+# This will exec the CMD passed to docker, i.e. "gunicorn" or "celery"
+exec "$@"
```
Comment (danlamanna) [inline analytics/entrypoint.sh:7]: Will this exec even if migrate fails?

**Review by danlamanna — APPROVED** [review]

**Pull request merged**